examples/binding-scala/js/src/main/scala/com/thoughtworks/todo/Main.scala

```diff
@@ -0,0 +1,148 @@
+package com.thoughtworks.todo
+
+import com.thoughtworks.binding.Binding._
+import com.thoughtworks.binding.dom
+import scala.collection.GenSeq
+import scala.scalajs.js.annotation.JSExport
+import org.scalajs.dom._
+import org.scalajs.dom.ext._
+import org.scalajs.dom.raw._
+import upickle.default._
+
+@JSExport object Main {
+
+  final class Todo(val title: String, val completed: Boolean)
+
+  object Todo {
```

Comment:
`Todo` is not a case class because we want to distinguish two `Todo`s with the same content.

If it is a `case class`, many `allTodos.get.indexOf(todo)` will be broken. e.g.
https://github.com/ThoughtWorksInc/todomvc/blob/8d9d8bde69d397c3fc8b010405c937e7fdaca3f1/examples/binding-scala/js/src/main/scala/com/thoughtworks/todo/Main.scala#L82